analyses/MA_cd_2010/03_sim_MA_cd_2010.R

```diff
@@ -7,7 +7,9 @@
 cli_process_start("Running simulations for {.pkg MA_cd_2010}")
 
 set.seed(2010)
-plans <- redist_smc(map, nsims = 2500, runs = 2L, counties = pseudo_county)
+plans <- redist_smc(map, nsims = 3500, runs = 2L, counties = county_muni) %>% match_numbers("cd_2010") %>% group_by(chain) %>%
```

Comment:
can you add line breaks after each `%>%`?

make sure also to run `enforce_style("MA")` one more time